bumpversion/cli.py

```diff
@@ -66,6 +66,46 @@
 ]
 
 
+def main(original_args=None):
+    defaults = {}
+    vcs_info = {}
+    part_configs = {}
+    files = []
+    new_version = None
+
+    args, known_args, root_parser, positionals = _parse_arguments_phase_1(original_args)
+    _setup_logging(known_args.list, known_args.verbose)
+    _determine_vcs_usability(VCS, vcs_info)
```

Comment:
I'm not a fan of passing in a "pointer". How about returning a dict with the info instead?

```suggestion
    vcs_info = _determine_vcs_usability(VCS)
```

Reply:
This was an artifact of my refactoring approach (PyCharm Extract->Method).  I'll fix the occurrences that I can find.